ha/core/system_health/system_health.py

```diff
@@ -86,6 +88,22 @@ def get_node_status(self, node_id, **kwargs):
         """
         get node status method. This method is for returning a status of a node.
         """
+        try:
+            # Prepare key and read the health value.
+            key = self._prepare_key(const.COMPONENTS.NODE_MAP.value, node_id=node_id)
+            node_map_val = self.healthmanager.get_key(key)
+            if node_map_val is not None:
+                node_map_dict = ast.literal_eval(node_map_val)
+                key = self._prepare_key(component='node', cluster_id=node_map_dict[COMPONENT_IDS.CLUSTER_ID.value],
+                                        site_id=node_map_dict[COMPONENT_IDS.SITE_ID.value], rack_id=node_map_dict[COMPONENT_IDS.RACK_ID.value],
+                                        storageset_id=node_map_dict[COMPONENT_IDS.STORAGESET_ID.value], node_id=node_id, **kwargs)
+                return self.healthmanager.get_key(key)
```

Comment:
Here we return the whole entity health dictionary.
@ajaysrivas Your comments please.

Reply:
we should get status related things only.
status and timestamp.

Reply:
Modified.
"get_node_status" returns dictionary of 'status' & 'created_timestamp' only.